content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md

```diff
@@ -0,0 +1,193 @@
+---
+title: "Create and Manage AWS IaaS Cluster"
+metaTitle: "Create and Manage AWS Cluster"
+metaDescription: "Learn how to add and manage a cluster deployed to AWS."
+hideToC: false
+fullWidth: false
+category: ["how-to"]
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+# Create and Manage AWS IaaS Cluster
+
+Palette supports creating and managing Kubernetes clusters deployed to an AWS account. This section guides you on how to create a Kubernetes cluster in AWS that is managed by Palette.
+
+# Prerequisites
+
+The following prerequisites must be met before deploying a cluster to AWS:
+
+- Access to an AWS cloud account 
+- Palette integration with AWS account. Review the [Add AWS Account](/clusters/public-cloud/aws/create-gov-accounts) for guidance.
+- An infrastructure cluster profile. Review the [Create Cluster Profiles](/cluster-profiles/task-define-profile) for guidance.
+- An [EC2 Key Pair](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html) in the target region.
+- Palette creates compute, network, and storage resources in AWS during the provisioning of Kubernetes clusters. Ensure there is sufficient capacity in the preferred AWS region for the creation of the following resources:
+    - vCPU
+    - VPC
+    - Elastic IP
+    - Internet Gateway
+    - Elastic Load Balancers
+    - NAT Gateway
+
+
+<InfoBox>
+
+
+The following tags should be added to the virtual private network (VPC) public subnets to enable automatic subnet discovery for integration with AWS load balancer service. Replace the value `yourClusterName` with your cluster's name.
+
+<br />
+
+- `kubernetes.io/role/elb = 1`
+- `sigs.k8s.io/cluster-api-provider-aws/role = public`
+- `kubernetes.io/cluster/[yourClusterName] = shared` 
+- `sigs.k8s.io/cluster-api-provider-aws/cluster/[yourClusterName] = owned`
+
+</InfoBox>
+
+
+# Deploying an AWS Cluster
+
+<!-- `video: title: "aws-cluster-creation": ./cluster-creation-videos/aws.mp4` -->
+
+Use the following steps to provision a new AWS cluster:
+
+1. Ensure you are in the correct project scope.
+
+
+2. Navigate to the left **Main Menu** and click on **Clusters**
+
+
+3. Click on **Add New Cluster**
+
+
+4. You will receive a prompt asking you if you want to deploy a new cluster or import an existing cluster. Click on **Deploy New Cluster**
+
+
+5. Select **AWS** and click on **Start AWS Configuration**
+
+
+6. Populate the wizard page with the following information: name, description, tags and AWS account. Tags on a cluster are propagated to the VMs deployed on the cloud/data center environments. Click on **Next** after you have filled out all the required information.
```

Comment:
`Populate the wizard page with the following information: name, description, tags, and AWS account. Tags on a cluster are propagated to the VMs deployed on the cloud/data center environments. Click on **Next** after completing all the required information.`